server.js

```diff
@@ -304,7 +304,7 @@ app.get('/answerForm', function(req, res){
                 answerForm:true});
             break;
         default:
-            return res.redirect('/');
+            return res.status(400).send('Please select an appropriate question Type.')
```

Comment:
Nice. I think there is another place for this. I will have to check my previous comments